Build/TestingSetupForWebTest.targets

```diff
@@ -30,8 +30,23 @@
     <Error Text="The property 'AppConfigFile' is not set." Condition="'$(AppConfigFile)' == ''" />
     <Error Text="The property 'Browser' is not set." Condition="'$(Browser)' == ''" />
 
+    <PropertyGroup>
+      <_dockerEnabled>False</_dockerEnabled>
+      <_dockerEnabled Condition="'$(DisableDocker)' == 'False'">True</_dockerEnabled>
+      <!-- Hostname can not be longer than 63 characters-->
+      <_dockerHostName>ReMotionWebTestContainer</_dockerHostName>
+      <_dockerPortNummer>60402</_dockerPortNummer>
+      <_dockerWebApplicationRoot>http://$(_dockerHostName):$(_dockerPortNummer)/</_dockerWebApplicationRoot>
+      <_dockerTimeout>00:15:00</_dockerTimeout>

```

Comment:
Yeah exactly. I originally proposed two timeouts, but that was denied. Overriding where exactly? Maybe MK has another idea when he reviews

Reply:
Downloading has the potential to take a long time (--> 15minutes, because of downloading a big image) and is the one which can fail periodically (server down, etc.). Starting the Container is usually fast, in some cases it could take a minute or so until everything is up and running and should rarely fail.

Reply:
So, basically, we should almost never have a build with a failed container start hanging for the timeout anyway, be it 2 minutes or 15. therefor, a second timeout would just optimize for the rare case by adding more complexity.